canonical-web-and-design/ubuntu.com
# Add copy global-nav for discourse

## Done
Add copy global-nav for discourse

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/static/js/build/global-nav.js
- Check you see the JS for global nav
